config/initializers/time.rb

```diff
@@ -1,7 +1,7 @@
 # Extends time class with convenience methods
 class Time
   def display_date
-    getlocal.strftime("%Y-%m-%d")
+    getlocal.strftime("%m-%d-%Y")
```

Comment:
Does not work, you should verify this. I have workarounds but would like a solution to keep tests consistent.